cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/IDs.kt

```diff
@@ -78,9 +78,14 @@ data class WireIdentity(
     val handle: String,
     val displayName: String,
     val domain: String,
-    val certificate: String
+    val certificate: String,
+    val status: DeviceStatus
 )
 
+enum class DeviceStatus {
```

Comment:
suggestion: since this is representing the certificate status, I think we can choose a better name here. wdyt?